src/components/BookForm.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { addBook, fetchBooks } from '../redux/Books/booksSlice';
+
+const BookForm = () => {
+  const categories = ['Action', 'ScienceFiction', 'Maths', 'Economy'];
+  const dispatch = useDispatch();
+
+  // Create category options for the select dropdown
+  const categoryOptions = categories.map((category, key) => (
+    // eslint-disable-next-line react/no-array-index-key
+    <option value={category} key={key}>
+      {category}
+    </option>
+  ));
+
+  // Initialize state for the book form inputs
+  const [book, setBook] = useState({
+    title: '',
+    author: '',
```

Comment:
I have noticed you disabled eslint in this file it is against microverse policy to disable linters beause linters help us write clean code it would be nice to fix this by enabling eslint in all files where it has been disabled

Reply:
done